project-koku/koku
# Limit schema update

Updating OCP report schema to align with Memory and CPU limits report data.
`pod_charge_cpu_cores => pod_limit_cpu_core_seconds`
`pod_limit_memory_bytes => pod_limit_memory_byte_seconds`

*Testing*
1.  Add OCP provider.
2. Add rates provider to get charge information
3. Upload usage data to upload service (This example is using real OCP data)
4. Download/Process data with masu
5. Verify Koku API OCP endpoints

*Test Results*
[limit_fix_ut.txt](https://github.com/project-koku/koku/files/2605124/limit_fix_ut.txt)
